### PR TITLE
New mock middleware

### PR DESCRIPTION
**Pull request by tobiasqueck**
Implementation for #532

### CODE REVIEW

File: packages/fiori-elements-writer/test/__snapshots__/alp.test.ts.snap [hunk 1]
```diff
@@ -127,11 +129,13 @@ server:
...
       configuration:
-        service:
-          urlPath: /sap/opu/odata/sap/SEPMRA_ALP_SO_ANA_SRV
-          metadataXmlPath: ./webapp/localService/metadata.xml
-          mockdataRootPath: ./webapp/localService/data
-          generateMockData: true
+        mountPath: /
+        services:
+          - urlPath: /sap/opu/odata/sap/SEPMRA_ALP_SO_ANA_SRV
+            metadataXmlPath: ./webapp/localService/metadata.xml
```
Comment (nlunets) [inline packages/fiori-elements-writer/test/__snapshots__/alp.test.ts.snap:135]: while you're at it you can rename metadataXmlPath to metadataPath and mockdataRootPath to mockdataPath :)

Comment (tobiasqueck) [reply packages/fiori-elements-writer/test/__snapshots__/alp.test.ts.snap]: Done with https://github.com/SAP/open-ux-tools/pull/533/commits/2f0d1de73bc031a6adf2df712175de8153fe5766

**Review by devinea — APPROVED** [review]
Hopefully the services config changes don't cause too much confusion.
I assume it is backward compatible to the old format also?

**Comment by tobiasqueck** [comment]
> I assume it is backward compatible to the old format also?

Yes, it is. I still prefer that we switch to the new one since it makes it easier to add another service to an existing project.

**Review by nlunets — APPROVED** [review]

**Pull request merged**